src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md

```diff
@@ -29,10 +29,10 @@ that not all uses of `wasm32-unknown-unknown` are using JavaScript and the web.
 
 ## Target maintainers
 
-When this target was added to the compiler platform-specific documentation here
+When this target was added to the compiler, platform-specific documentation here
 was not maintained at that time. This means that the list below is not
-exhaustive and there are more interested parties in this target. That being
-said since when this document was last updated those interested in maintaining
+exhaustive, and there are more interested parties in this target. That being
+said, when this document was last updated, those interested in maintaining
 this target are:
```

Comment:
it seems it's harder to understand with `,` (I'm not a native English speaker)

Reply:
we can remove "when this document was last updated", such that we have

> That being said, those interested in maintaining this target...

Reply:
yeah, make sense.